### ISSUE TITLE
Results and discussion, first three sections

### ISSUE THREAD

**Issue by ohsOllila** [hsantila/Corrtimes#29]
Here are some thoughts on the first three sections in Results and Discussion section. I would reformulate the sections and slightly change the order of the content. I am ready to draft the reorganized text. I estimate that it takes a few hours of work. 

1) I would merge the "introduction" to the subsections. I think that we can modify the subtitles such that reader gets the essential information on the content of the Results and Discussion just looking the titles with need for the introduction listing the content.

2) I would merge the sections "Effective correlation times τ e at standard conditions." and "R 1 rates at standard conditions." into a new section titled "Internal lipid dynamics in POPC bilayer compared with NMR experiments" or something like that. I have a somekind of idea how such section could be written, but it would be easiest to describe this by drafting such section based on the current text. In any case I would avoid using the term "Standard conditions", at least in titles.

3) I would retitle the section "Dynamics of headgroup segments in CHARMM36." to "Internal motions in POPC headgroup and glycerol backbone" or something like that, and I would reformulate the discussion to more general discussion on lipid dynamics, away from specific discussion on different force fields. This is again difficult to explain without drafting.

**Comment by markussmiettinen** [comment]
> Here are some thoughts on the first three sections in Results and Discussion section. I would reformulate the sections and slightly change the order of the content. I am ready to draft the reorganized text. I estimate that it takes a few hours of work.

It is a bit unclear to me if these will improve the manuscript. Thus, maybe you should indeed prepare a suggestion. (I am sorry that I proposed this issue-route first, but maybe we can have the discussion here, anyways.)

> 1. I would merge the "introduction" to the subsections. I think that we can modify the subtitles such that reader gets the essential information on the content of the Results and Discussion just looking the titles with need for the introduction listing the content.

I think the first paragraph of 'introduction' is really not needed. The second I would keep, and have it before discussing the results, because it underlines why it does not make sense to go very detailed analysis of structure or dynamics of the models.

> 2. I would merge the sections "Effective correlation times τ e at standard conditions." and "R 1 rates at standard conditions." into a new section titled "Internal lipid dynamics in POPC bilayer compared with NMR experiments" or something like that. I have a somekind of idea how such section could be written, but it would be easiest to describe this by drafting such section based on the current text. In any case I would avoid using the term "Standard conditions", at least in titles.

I don't see how such a combined section could be clearer to the reader.

I would be willing to replace 'standard conditions' with some other term, should we find such. However, why do you find this term misleading?

> 3. I would retitle the section "Dynamics of headgroup segments in CHARMM36." to "Internal motions in POPC headgroup and glycerol backbone" or something like that, and I would reformulate the discussion to more general discussion on lipid dynamics, away from specific discussion on different force fields. This is again difficult to explain without drafting.

I am afraid on going to discussion of real lipid dynamics because the force fields do not reproduce the order parameters, and because I think our data analysis should be better do that.

**Comment by ohsOllila** [comment]
I have now made a pull request with the suggestion to merge the first two sections. I did this into a another branch, however, it seems that I did it in the way that it does not actually allow separate acceptance of this part, sorry about this. (offtopic note: My experience is nowadays that overleaf works for writing process so much better that transferring even this manuscript there might be worth of doing. A commercial lincense is needed thought.). Let me know if you want that I do some reorganization for the pull requests.

>I think the first paragraph of 'introduction' is really not needed. The second I would keep, and have it before discussing the >results, because it underlines why it does not make sense to go very detailed analysis of structure or dynamics of the models.

The second paragraph is now in the new location.

>I would be willing to replace 'standard conditions' with some other term, should we find such. However, why do you find this term >misleading?

I think that "standard conditions" is not very appealing and self-explanatory title. I think that there are very few people who would be tempted to read a piece of text titled "R 1 rates at standard conditions." However, "Internal dynamics of POPC lipids" or something like that describes in more general terms what is inside the section.

I will try to push a suggestion for the third section today, and more minor suggestions and comments for the rest of the manuscript before vacation.
